### PR TITLE
[fix bug 1104629] Do not resolve newly created action items.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: remo/remozilla/models.py [hunk 1]
```diff
@@ -127,26 +123,36 @@ def save(self, *args, **kwargs):
...
+                possible_actions = [ADD_RECEIPTS_ACTION, ADD_REPORT_ACTION,
+                                    ADD_PHOTOS_ACTION,
+                                    ADD_REPORTS_PHOTOS_ACTION,
+                                    REVIEW_BUDGET_REQUEST_ACTION,
+                                    WAITING_MENTOR_VALIDATION_ACTION]
                 action_names = (['{0} {1}'.format(action, self.summary)
-                                 for action in actions])
-
+                                 for action in possible_actions])
+                bug_attrs = ['waiting_receipts', 'waiting_report',
```
Comment (johngian) [inline remo/remozilla/models.py]: I suggest you move this outside this class and use it as a constant. It's probably better to be under "remozilla" app.

File: remo/remozilla/models.py [hunk 2]
```diff
@@ -127,26 +123,36 @@ def save(self, *args, **kwargs):
...
-                items = ActionItem.objects.filter(content_type=action_model,
-                                                  object_id=self.pk)
-                items.update(resolved=True)
+                action_items.update(resolved=True)
             else:
-                actions = [ADD_RECEIPTS_ACTION, ADD_REPORT_ACTION,
-                           ADD_PHOTOS_ACTION, ADD_REPORTS_PHOTOS_ACTION,
-                           REVIEW_BUDGET_REQUEST_ACTION,
-                           WAITING_MENTOR_VALIDATION_ACTION]
+                possible_actions = [ADD_RECEIPTS_ACTION, ADD_REPORT_ACTION,
```
Comment (johngian) [inline remo/remozilla/models.py]: Same here.

File: remo/remozilla/models.py (hunk 1 above)
Comment (johngian) [inline remo/remozilla/models.py]: I would prefer this to be a constant.

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
lgtm r+

**Pull request merged**